### ISSUE TITLE
Incredibly long start-up times

### ISSUE THREAD

**Issue by GFMB** [CatastropheModpack/Gregtech6-for-GT5u#2]
I don't know what exactly is causing this, but even GTNH is starting up faster. It sometimes takes up to 40 minutes just to start, and then it crashes. And the fact that it is registering 80 000 materials also seems weird.

**Comment by monatann** [comment]
1. This mod needs my modified GTNH's GregTech5u version and modified dependences files such as GTNH Core Mod.
GTNH mods are very special so it causes crash if GTNH changes codes of many mods.

2. The pack I am making is about 490 mods, and it takes 4+ hours to start-up.

3. I just re-named GT6 package for anti-conflict, and I only change a few codes.
There are GT5U materials and GT6 materials, so the amount of materials is not weird.
There is the only way to re-write sooo many codes.

In conclusion,
1. There is not crash report so I can't help.
2. Need re-written codes if we want to play comfortably.

**Comment by GFMB** [comment]
Oh yeah sorry, I forgot sending the crash report. Here it is: https://pastebin.com/emCmQ1Na
I used normal GT5U so that may be the problem. I take it that the mods needed are these? https://github.com/CatastropheModpack/GTNH-Dependencies-Mods
I forgot to add that I also used GT++. I heard that this was the only GT5U+GT6 fusion mod which allowed for GT5(U) addons to be used, hence why I downloaded it.

**Comment by monatann** [comment]
Well...is it repeat-able problem?
And have you tried to remove GT6 and cause same issue?
According to crash report, I think GT++ tried to replace biome and  EnhancedBiomes tried too, then caused crash.

at gtPlusPlus.australia.chunk.ChunkProviderAustralia.func_147422_a(ChunkProviderAustralia.java:219)
->
at enhancedbiomes.handlers.ReplaceBiomeBlocksHandler.replaceBlocksForBiome(ReplaceBiomeBlocksHandler.java:42)
->
NullPointerException

Caused by "replaceBlocksForBiome" section so it's related with something about Biome.

**Comment by monatann** [comment]
![image](https://user-images.githubusercontent.com/41532598/64285219-4ae3d500-cf96-11e9-9e4c-507b60fc7298.png)
![image](https://user-images.githubusercontent.com/41532598/64285226-50d9b600-cf96-11e9-8eab-fd1103b8bfb9.png)
![image](https://user-images.githubusercontent.com/41532598/64285242-56370080-cf96-11e9-8ccc-997a57e4fe90.png)
![image](https://user-images.githubusercontent.com/41532598/64285251-5afbb480-cf96-11e9-967f-19b67e9c0765.png)

Maybe GT6 do nothing about that crash.

**Comment by GFMB** [comment]
Could be just GT++, then. I am just a bit confused since it says something about Australia and that's another dimension GT++ adds. I don't know why they'd conflict unless GT++ Australia adds overworld biomes. I will try it again soon, I simply didn't do that right away due to how long it took to start in the first place. And, while I don't know whether that's relevant or not, even before adding the fusion mod, GT++, and so on, the worldgen was pretty weird. The worlds sometimes didn't even generate at all and then, sometimes, when I got them to generate, I couldn't even explore the world properly as it ended a couple of chunks from spawn already. I kind of fixed that now and I'll try it out with GT++ et cetera now.

**Comment by monatann** [comment]
Well...is GT++bad?
I think enhancedbiomes catch "null" and cause crash, so enhancedbiomes is bad, not GT++.

**Comment by GFMB** [comment]
Yeah, it works now. I removed enhanced biomes instead of GT++ because I'd rather have GT++ work than that. Also turns out that enhanced biomes was the mod which caused worldgen to take ages.
